packages/cosmic-swingset/lib/ag-solo/start.js

```diff
@@ -282,7 +286,7 @@ export default async function start(basedir, withSES, argv) {
   // Start timer here!
   startTimer(1200);
 
-  console.log(`swingset running`);
+  log.info(`swingset running`);
```

Comment:
This one might be helpful. Does anything come later to tell you it's running? I'd make it 
```
log('swingset running');
```

Reply:
`log.info()` is higher priority than `log()`.